content/api-docs/entry/alerts.md

```diff
@@ -101,9 +101,9 @@ The designated alert will be closed.
 
 ### Input
 
-```javascript
+```json
```

Comment:
I noticed the Japanese document uses `json` while the English document uses `javascript` .
I assume `json` is preferable here.

https://mackerel.io/api-docs/entry/alerts
https://mackerel.io/ja/api-docs/entry/alerts

Reply:
I made a separate pull request to fix all the occurrences of `javascript` :
https://github.com/mackerelio/documents/pull/6